---
title: Errore del compilatore CS1017
ms.date: 07/20/2015
ms.prod: .net
ms.technology:
- devlang-csharp
ms.topic: article
f1_keywords:
- CS1017
helpviewer_keywords:
- CS1017
ms.assetid: e0902e8a-b042-4711-a8a6-83456a3f88cb
caps.latest.revision: 
author: BillWagner
ms.author: wiwagn
ms.openlocfilehash: adfebf5f696090aa7355025e4c2cba2240e1a6c3
ms.sourcegitcommit: 83dd5ec003e788ccb3e33f3412a7af39ae347646
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 03/15/2018
---
# <a name="compiler-error-cs1017"></a>Errore del compilatore CS1017
Le clausole catch non possono seguire la clausola catch generale di un'istruzione try  
  
 Un blocco `catch` che non accetta parametri deve essere l'ultimo in una serie di blocchi `catch` . Per altre informazioni sulle eccezioni, vedere [Exception Handling Statements](../../csharp/language-reference/keywords/exception-handling-statements.md).  
  
## <a name="example"></a>Esempio  
 L'esempio seguente genera l'errore CS1017:  
  
```csharp  
// CS1017.cs  
using System;  
  
namespace x  
{  
    public class b : Exception  
    {  
    }  
  
    public class a  
    {  
        public static void Main()  
        {  
            try  
            {  
            }  
  
            catch   // CS1017, must be last catch  
            {  
            }  
  
            catch(b)  
            {  
                throw;  
            }  
        }  
    }  
}  
```
